bentoml/frameworks/pytorch.py

```diff
@@ -75,7 +76,8 @@ def pack(self, model, metadata=None):  # pylint:disable=arguments-differ
 
         if not isinstance(model, torch.nn.Module):
             raise InvalidArgument(
-                "PytorchModelArtifact can only pack type 'torch.nn.Module'"
+                "PytorchModelArtifact can only pack type \
```

Comment:
Could you add a usage example in the docstring above in this class?

Reply:
Thanks for the suggestion :D
Updated accordingly!